packages/api/src/schema/schema.yaml

```
@@ -3294,6 +3358,25 @@ paths:
     post:
       description: create a new room
       responses:
+        200:
+          description: Success
+          content:
+            application/json:
+              $ref: "#/components/schemas/create-room-response"
```

If it's only for $refing it here, I'd say we can just inline it (and remove the top-level component)
```suggestion
              additionalProperties: false
              properties:
                id:
                  type: string
```